src/katsdpcontroller/fake_servers.py

```diff
@@ -294,7 +344,7 @@ def __init__(self, *args, **kwargs) -> None:
                 Sensor(
                     str,
                     f"{corrprod_name}.chan-range",
-                    "The range of channels processed by this XB-engine, inclusive",
+                    "The range of channels processed by this X-engine, inclusive",
```

Comment:
Why this change?

Reply:
This description now matches the ICD exactly (as does the equivalent B-engine sensor).